eng/Publishing.props

```diff
@@ -142,13 +142,13 @@
   <PropertyGroup>
       <!--
         Because we may be building in a container, we should use an asset manifest file path that exists in the container. Disambiguate the manifests via available properties.
-        AGENT_OS and AGENT_JOBNAME are present on Azure DevOps agents. AssetManifestOS will also be used by arcade to generate the name of the manifest file name for the built in publishing.
+        AGENT_OS and SYSTEM_PHASENAME are present on Azure DevOps agents. AssetManifestOS will also be used by arcade to generate the name of the manifest file name for the built in publishing.
```

Comment:
`AGENT_JOBNAME` ends up grabbing the display name of the job, which now contains invalid characters for file creation. `SYSTEM_PHASENAME` is the actual correct job name. This is because of legacy reasons in Azure Pipeline's design. See: https://developercommunity.visualstudio.com/t/systemjobname-seems-to-be-incorrectly-assigned-and/1209736#TPIN-N1211828